docs/content/en/docs/getting-started/production-environment/_index.md

```diff
@@ -19,7 +19,7 @@ The alternative is to simply use your initial cluster to run workloads.
 {{% alert title="Important" color="warning" %}}
 
 Creating an EKS Anywhere management cluster is the recommended model.
-In its first release, EKS Anywhere clusters functioned as standalone clusters.
+In its first release, EKS Anywhere clusters functioned as self-managed clusters.
```

Comment:
Do we even want to call this out? If docs are versioned I'm not sure this is important information to include.

Reply:
I removed the sentence.